addons/point_of_sale/static/src/app/generic_components/product_card/product_card.xml

```diff
@@ -11,6 +11,7 @@
             <div t-if="props.imageUrl" class="product-img">
                 <img class="w-100" t-att-src="props.imageUrl" t-att-alt="props.name" />
             </div>
+            <div t-else="" class="no-product-image" />
```

Comment:
you can remove the `t-if` from the first div and put it on the `img` inside of it. This way you don't need the second `div` and the new css  class

Reply:
I need the second class to have a minimum height when there's not image. Otherwise the product name is not aligned correctly. I changed to this: 

```
 <div t-attf-class="{{ props.imageUrl ? 'product-img' : 'no-product-image' }}">
      <img class="w-100" t-if="props.imageUrl" t-att-src="props.imageUrl" t-att-alt="props.name" />
  </div>
```